src/logic/powerflowresults.py

```diff
@@ -178,3 +179,32 @@ def report_infeasible(self):
                 results.append(gen_result)
 
         return results
+
+    def output(self, voltagefilepath:Path=Path(".output", "voltage.txt"), powerfilepath:Path=Path(".output", "power.txt")):
```

Comment:
Ah, write to file. Excellent. If it's not too much more, maybe we should write this as a CSV? I suspect that'll make it a more digestible format.